word_embedding/predict_v2.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
@lcy-seso agree, how about rename predict_v2.py to hsigmoid_predict.py and train_v2.py to hsigmoid_train.py